ui/react-components/translations/errorTranslations.js

```diff
@@ -0,0 +1,31 @@
+export const getErrorTranslations = intl => {
```

Comment:
Any reason why we created a translations folder instead of having it in util??

Reply:
Nothing in particular. Later we thought to move normal translations also here. We can have both of them as util. So changed and moved